### PR TITLE
[ASA-1004][GW] update subscription complete page with agent name

### PR DESCRIPTION
**Pull request by geoffreywatson**

### CODE REVIEW

File: it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala [hunk 1]
```diff
@@ -152,32 +154,22 @@ trait SubscriptionControllerISpec extends BaseISpec with SessionDataMissingSpec
...
-
-    "contain a button to continue journey" when {
-      "a continue URL exists in the session, show a generic 'Continue' button using that URL" in new RequestWithSessionDetails {
-        val continueUrl = ContinueUrl("/test-continue-url")
-        sessionStoreService.currentSession.continueUrl = Some(continueUrl)
-
-        resultOf(request) should containSubstrings(s">${htmlEscapedMessage("subscriptionComplete.button.continueJourney")}</a>", continueUrl.url)
-      }
-
-      "no continue URL exists in the session, show a button with a link in AS services" in new RequestWithSessionDetails {
```
Comment (sureshhmrc) [inline it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala]: don't we need the above tests ?

File: conf/messages [hunk 2]
```diff
@@ -257,12 +257,16 @@ linkClients.error.no-radio-selected= Select yes if you want to link your clients
...
+subscriptionComplete.accountName=You created an account for
 subscriptionComplete.h2=What you can do next
+subscriptionComplete.h3=Help us improve this service
 subscriptionComplete.p1=Make a note of your account reference number <strong class="bold">{0}</strong>. You will need this if you are not able to access your agent services account.
-subscriptionComplete.p2=<a href="{0}">Check the guidance</a> to find out how to:
-subscriptionComplete.bullet-list.1=sign in to your agent services account
-subscriptionComplete.bullet-list.2=use new HMRC services on behalf of your clients
+subscriptionComplete.p2=You can:
+subscriptionComplete.p3=What did you think of this service?
+subscriptionComplete.p4= (takes 30 seconds)
```
Comment (sureshhmrc) [inline conf/messages]: is space ok after = ?

Comment (geoffreywatson) [reply conf/messages]: Yes we need the space to render the view correctly.

File: it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala (hunk 1 above)
Comment (geoffreywatson) [reply it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala]: Those elements have been removed from the page so no we don't need the tests

**Pull request merged**